web-lib-funcs.pl

```diff
@@ -4449,10 +4450,16 @@ =head2 get_webmin_version
 =cut
 sub get_webmin_version
```

Comment:
What does this change do?

Reply:
It removes dots from the version.

Reply:
But what if existing callers expect to get the dots in the version?

Reply:
So it looks like this actually caused a problem in the end :-( I think you should undo this, and make the caller remove dots from the version if needed..

Reply:
Sadly, but this was because the theme called this sub incorrectly, passing `1` as a parameter.

I made it happen in the caller https://github.com/webmin/webmin/commit/d7de842f450ff4c25f79e76d255965ecb7fdf474.